src/tools/clippy/tests/ui/transmutes_expressible_as_ptr_casts.rs

```diff
@@ -0,0 +1,76 @@
+#![warn(clippy::transmutes_expressible_as_ptr_casts)]
```

Comment:
```suggestion
// run-rustfix
#![warn(clippy::transmutes_expressible_as_ptr_casts)]
```

I think the suggestion of this lint is auto applicable, so we should test it.